src/sentry/tasks/unmerge.py

```diff
@@ -216,7 +216,7 @@ def migrate_events(
 
     event_id_set = set(event.id for event in events)
 
-    Event.objects.filter(project_id=project.id, id__in=event_id_set).update(group_id=destination_id)
+    # Event.objects.filter(project_id=project.id, id__in=event_id_set).update(group_id=destination_id)
```

Comment:
I was debating whether to remove this line in this PR as well. I think there's no reason not to though, since we don't rely on the Event group_id field being correct anymore